test/OpenFeature.Contrib.Providers.Flagd.Test/CacheTest.cs

```diff
@@ -0,0 +1,117 @@
+using Xunit;
+
+namespace OpenFeature.Contrib.Providers.Flagd.Test
+{
+    public class UnitTestLRUCache
```

Comment:
You may want to add one very simple test/retrieve test. Of course, this case is indirectly tested by the other cases, but without that simple case it might be confusing if there's a regression in that simple funcitonaltiy.